lib/cache.coffee

```diff
@@ -3,16 +3,32 @@ _ = require 'underscore'
 redis = require 'redis'
 @client = null
 
+retry_strategy = (options) ->
+  console.log('Retrying with', options)
+
+  if options.error and options.error.code == 'ECONNREFUSED'
+    # End reconnecting on a specific error and flush all commands with
+    # a individual error
+    return new Error('The server refused the connection')
+
+  if options.total_retry_time > 1000 * 60 * 60
+    # End reconnecting after a specific timeout and flush all commands
+    # with a individual error
+    return new Error('Retry time exhausted')
+
+  if options.attempt > 10
+    # End reconnecting with built in error
+    return undefined
+
+  # Reconnect after
+  Math.min options.attempt * 100, 3000
```

Comment:
Default retry strategy pulled from the redis readme here: https://github.com/NodeRedis/node_redis/blob/master/README.md

Seems like a reasonable default.